products/nexus.md

```diff
@@ -18,30 +20,356 @@ auto:
+
+-   releaseCycle: "3.41"
+    releaseDate: 2022-07-28
+    eoas: 2023-08-03
+    eol: 2024-01-27
+    latest: "3.41.1-01"
+    latestReleaseDate: 2022-08-19
+
+-   releaseCycle: "3.40"
+    releaseDate: 2022-06-22
+    eoas: 2023-08-03
+    eol: 2023-12-22
+    latest: "3.40.1-01"
+    latestReleaseDate: 2022-06-27
+
+-   releaseCycle: "3.39"
+    releaseDate: 2022-05-23
+    eoas: 2023-08-03
+    eol: 2023-11-19
+    latest: "3.39.0-01"
+    latestReleaseDate: 2022-05-23
+
+-   releaseCycle: "3.38"
+    releaseDate: 2022-03-03
+    eoas: 2023-08-03
+    eol: 2023-09-25
+    latest: "3.38.1-01"
+    latestReleaseDate: 2022-03-29
+
+-   releaseCycle: "3.37"
+    releaseDate: 2021-11-29
+    eoas: false
+    eol: 2023-05-24
+    latest: "3.37.3-02"
+    latestReleaseDate: 2021-12-29
+
+-   releaseCycle: "3.36"
+    releaseDate: 2021-10-29
+    eoas: false
+    eol: 2023-04-29
+    latest: "3.36.0-01"
+    latestReleaseDate: 2021-10-29
+
+-   releaseCycle: "3.35"
+    releaseDate: 2021-10-12
+    eoas: false
+    eol: 2023-04-13
+    latest: "3.35.0-02"
+    latestReleaseDate: 2021-10-12
+
+-   releaseCycle: "3.34"
+    releaseDate: 2021-09-01
+    eoas: false
+    eol: 2023-03-03
+    latest: "3.34.1-01"
+    latestReleaseDate: 2021-09-22
+
+-   releaseCycle: "3.33"
+    releaseDate: 2021-08-04
+    eoas: false
+    eol: 2023-02-04
+    latest: "3.33.1-01"
+    latestReleaseDate: 2021-08-18
+
+-   releaseCycle: "3.32"
+    releaseDate: 2021-07-08
+    eoas: false
+    eol: 2023-01-08
+    latest: "3.32.1"
+    latestReleaseDate: 2021-12-20
+
+-   releaseCycle: "3.30"
+    releaseDate: 2021-03-04
+    eoas: false
+    eol: 2022-09-04
+    latest: "3.30.1-01"
+    latestReleaseDate: 2021-04-22
+
+-   releaseCycle: "3.29"
+    releaseDate: 2020-12-04
+    eoas: false
+    eol: 2022-06-04
+    latest: "3.29.2-02"
+    latestReleaseDate: 2021-01-07
+
+# From https://help.sonatype.com/docs/sonatype-sunsetting-information/sonatype-nexus-repository-2-versions-status
+
+-   releaseCycle: "2.15"
+    releaseDate: 2022-03-17
+    eoas: true
     eol: false
     latest: "2.15.1-02"
     latestReleaseDate: 2022-03-23
-    link: https://help.sonatype.com/repomanager2/release-notes
+
+-   releaseCycle: "2.14"
+    releaseDate: 2016-09-20
+    eoas: true
+    eol: false
+    latest: "2.14.21-02"
+    latestReleaseDate: 2021-12-20
+
+-   releaseCycle: "2.13"
+    releaseDate: 2016-04-18
+    eoas: false
+    eol: 2017-10-17
+    latest: "2.13.0"
+    latestReleaseDate: 2016-04-18
 
 ---
 
-> [Nexus Repository OSS](https://help.sonatype.com/repomanager3/product-information/download) is an open
-> source repository manager developed by Sonatype that supports many artifact formats, including
+> [Nexus Repository](https://help.sonatype.com/repomanager3) is a
+> repository manager developed by Sonatype that supports many artifact formats, including
 > Docker, Java, and npm.
+> It comes in OSS (open source) and Pro (commercial) versions.
+
+Lifecycle
```

Comment:
I think the description can be shortened to something link this:

```
All minor releases are typically supported for 18 months. This includes 12 months of full support,
with updates and bug fixes, followed by 6 months of extended maintenance, with only critical and
security fixes. After this, the release is considered sunset.

Sonatype provides [commercial support](https://www.sonatype.com/usage/software-support-policy) with
additional features with [Nexus Repository Pro](https://www.sonatype.com/products/sonatype-nexus-oss-vs-pro-features).
```

This is more in line with the way products are describes on endoflife.date (for example: https://endoflife.date/angular), and it does not contains unknown terms such as PDLC while describing the main ones (full support, extended support and sunset).

WDYT ?

Reply:
Totally agree is way clearer like that. Changes performed.